RecoBTau/JetTagComputer/src/GenericMVAJetTagComputer.cc

```diff
@@ -1,8 +1,10 @@
 #include <algorithm>
 #include <iostream>
-#include <string>
 #include <memory>
```

Comment:
duplicate

Reply:
hm it seems this substitution to make_unique includes <memory> regardless was it already included